src/app/js/fields/FilterAt.js

```diff
@@ -1,16 +1,18 @@
 import * as React from 'react';
 import SvgIcon from '@material-ui/core/SvgIcon';
 
-export const FilterAt = props => {
+export const FilterAt = React.forwardRef((props, ref) => {
```

Comment:
What is the `ref` for ?

Reply:
Mui Tooltip is not working properly with my custom icon `FilterAt`. It's because the child of a Material-UI Tooltip must be able to hold a ref. When i googled my probleme, I saw that using React.forwardRef would solve my problem

Reply:
Good to know! :)